evm-contracts/src/v0.6/dev/FluxAggregator.sol

```diff
@@ -498,7 +498,7 @@ contract FluxAggregator is AggregatorInterface, Owned {
   function latestSubmission(address _oracle)
     external
     view
-    returns (int256, uint256)
+    returns (int256, uint32)
```

Comment:
The AggregatorInterface treats this type as a `uint256`. Is there any benefit to not keeping that consistent in the external API?